src/components/html-renderer/html-renderer.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { renderMarkdownToHTML } from './utils';
+import './html-renderer.scss';
+
+const HTMLRenderer = ({ content, className = '', fontSize }) => {
+  const style = fontSize ? { fontSize } : {};
+
+  return (
+    <div
+      className={`html-renderer ${className}`}
+      style={style}
+      dangerouslySetInnerHTML={{
```

Comment:
Since we can’t guarantee what user-authored content will contain, using dangerouslySetInnerHTML without sanitization is risky. We should switch to react-markdown or another alternative?